### PR TITLE
Fixing descriptions and labels

### PR DESCRIPTION
**Pull request by peterroth**

### CODE REVIEW

File: app/src/ui-blokada/res/values/strings_tunnel.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <string name="tunnel_config_wifi_only_title">Download on WiFi only</string>
     <string name="tunnel_config_wifi_only_description">Activate this switch if you wish the app to download lists only when your device is connected to WiFi. Otherwise, the app may also download lists when your device is connected to a mobile network and use your data plan.</string>
     <string name="tunnel_config_watchdog_title">Watchdog</string>
-    <string name="tunnel_config_watchdog_description">Activate this switch if you wish the app to activate the VPN only if there is Internet connectivity, and restart the tunnel in case there are connectivity problems. This may help for some devices. Note: in earlier versions this option was called "Online Only".</string>
```
Comment (kar) [inline app/src/ui-blokada/res/values/strings_tunnel.xml]: nobody remembers anymore? :D

File: app/src/ui-blokada/res/values/strings_bits.xml [hunk 2]
```diff
@@ -89,7 +89,7 @@
     <string name="menu_vpn_support_label">If you experience any problems, do not hesitate to contact us.</string>
     <string name="menu_vpn_support_button">Support</string>
     <string name="menu_vpn_tunnel_settings">Blokada Tunnel settings</string>
-    <string name="menu_vpn_tunnel_learn_more">Learn mone about Blokada Tunnel</string>
+    <string name="menu_vpn_tunnel_learn_more">Learn more about Blokada Tunnel</string>
```
[the remark this answers is not in the document]
Comment (peterroth) [reply app/src/ui-blokada/res/values/strings_bits.xml]: Change reverted.

File: app/src/ui-blokada/res/values/strings_dns.xml [hunk 3]
```diff
@@ -6,8 +6,8 @@
 
     <!-- An example format of a DNS server address. -->
     <string name="dns_edit_ip_label">IP address</string>
-    <string name="dns_edit_ip1_enter">Enter a DNS server address</string>
-    <string name="dns_edit_ip2_enter">Enter a secondary DNS server address</string>
+    <string name="dns_edit_ip1_enter">Enter DNS server's primary address</string>
```
[the remark this answers is not in the document]
Comment (peterroth) [reply app/src/ui-blokada/res/values/strings_dns.xml]: Description shortened, apostrophes removed.

File: app/src/ui-blokada/res/values/strings_panel.xml [hunk 4]
```diff
@@ -190,7 +190,7 @@
 
     <string name="subscription_success">Your account is being activated. This may take a few seconds. Thank you!</string>
 
-    <string name="slot_start_on_boot_description">Enabling start on boot allows Blokada to be active at all times and protect your device.</string>
+    <string name="slot_start_on_boot_description">Enabling start on boot allows Blokada to be active at all times and protect your device. Please note the system always has higher priority and can overwrite this option, thus, if Blokada doesn't start up, take a look in your system's settings.</string>
```
[the remark this answers is not in the document]
Comment (peterroth) [reply app/src/ui-blokada/res/values/strings_panel.xml]: Escaped.

File: app/src/ui-blokada/res/values/strings_tunnel.xml (hunk 1 above)
Comment (peterroth) [reply app/src/ui-blokada/res/values/strings_tunnel.xml]: What was the last time is was called "Online Only"? 2.8? :)

**Pull request merged**